docs/runbooks/VirtHandlerDaemonSetRolloutFailing.md

```diff
@@ -1,23 +1,31 @@
 # VirtHandlerDaemonSetRolloutFailing
+<!--apinnick, Nov 2022-->
 
 ## Meaning
 
-Some virt-handler DaemonSet fail to roll out after 15 minutes. This alert suggests that in the cluster, at least one worker node does not have the virt-handler DaemonSet pod successfully rolled out in the given time. 
+The `virt-handler` daemon set has failed to deploy on one or more worker nodes after 15 minutes.
```

Comment:
@apinnick I believe it should be: daemon set -> DeamonSet also in additional places in this runbook

Reply:
[Downstream documentation conventions](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-general-references): When a Kubernetes object is used as a general reference ("The daemon set fails to deploy"), and not as an object reference ("Edit the `VirtualMachine` manifest"), we spell it out in lower case, with spaces.